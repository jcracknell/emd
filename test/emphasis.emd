*emphasized text*

emphasized *across
lines*

no*space*

missing *end

nested **strong *emphasis***

mismatched *emphasis **strong* end**

mismatched **strong *emphasis** end*
