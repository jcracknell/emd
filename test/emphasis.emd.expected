#! emd.cs.Conversion.StructuralRenderer, emd.cs
(Document
	(Paragraph
		(Emphasis
			(Text "emphasized")
			(Space)
			(Text "text")
		)
	)
	(Paragraph
		(Text "emphasized")
		(Space)
		(Emphasis
			(Text "across")
			(Space)
			(Text "lines")
		)
	)
	(Paragraph
		(Text "no")
		(Emphasis
			(Text "space")
		)
	)
	(Paragraph
		(Text "missing")
		(Space)
		(Emphasis
			(Text "end")
		)
	)	
	(Paragraph
		(Text "nested")
		(Space)
		(Strong
			(Text "strong")
			(Space)
			(Emphasis
				(Text "emphasis")
			)
		)
	)
	(Paragraph
		(Text "mismatched")
		(Space)
		(Emphasis
			(Text "emphasis")
			(Space)
			(Strong
				(Text "strong")
			)
		)
		(Space)
		(Text "end")
		(Strong
		)	
	)
	(Paragraph
		(Text "mismatched")
		(Space)
		(Strong
			(Text "strong")
			(Space)
			(Emphasis
				(Text "emphasis")
			)
		)
		(Space)
		(Text "end")
		(Emphasis
		)	
	)	
)
